components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/pages/src/app/components/api_listing/ApisTable.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react'
+import ReactDom from 'react-dom'
+
+class ApiTable extends Component {
+
+    render() {
+        const api_rows = [];
+        for (let api of this.props.apis.list) {
+            console.log(api);
+            api_rows.push(
+                <tr key={api.id}>
+                    <td>{api.name}</td>
+                    <td>/{api.context}</td>
+                    <td>{api.version}</td>
+                </tr>
+            );
+        }
+        return (
+            <div>
+                <table style={{width: '100%'}}>
+                    <tbody>
+                    <tr>
+                        <th>Name</th>
```

Comment:
thanks @anuruddhal for pointing out i18n work, There are several libraries[[1](https://github.com/yahoo/react-intl)][[2](https://github.com/i18next/react-i18next)] for react to make internationalization purpose, Will test them and integrate one of them as suitable for us.

[1] : https://github.com/yahoo/react-intl
[2] : https://github.com/i18next/react-i18next